karate-core/src/main/java/com/intuit/karate/Runner.java

```diff
@@ -67,7 +67,7 @@
         String scenarioName;
         List<String> tags = new ArrayList();
         List<String> paths = new ArrayList();
-        List<Resource> resources;
+        List<Resource> resources= new ArrayList<>();
```

Comment:
sorry to nitpick. just to follow the existing standards - a) space before the `=` and b) no need for the `<>`